astroquery/irsa/tests/test_irsa_remote.py

```diff
@@ -1,20 +1,20 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from __future__ import print_function
 
-from astropy.tests.helper import remote_data
+import pytest
+import astropy.units as u
 from astropy.table import Table
 from astropy.coordinates import SkyCoord
-import astropy.units as u
 
 from ... import irsa
 
 
 OBJ_LIST = ["m31", "00h42m44.330s +41d16m07.50s",
-            SkyCoord(l=121.1743, b=-21.5733, unit=(u.deg, u.deg),
+            SkyCoord(l=121.1743, b=-21.5733, unit=(u.deg, u.deg),  # noqa
```

Comment:
what was the QA issue here? (just curious)

Reply:
It complains about ambiguous variable. `flake8` doesn't like the`l`.

Reply:
I can back it out if you don't want it.

Reply:
oh, weird that it looks into class signatures when we have absolutely no control over what the upstream calls their args/kwargs.